# This config file is used by traj_demo to generate a trajectory
joint_names: ["fr3_joint1", "fr3_joint2", "fr3_joint3", "fr3_joint4", "fr3_joint5", "fr3_joint6", "fr3_joint7"]
waypoints:
  cartesion: # meters
    - position: [0.4, 0.0, 0.001]
      time: 1.0